telegram/ext/regexhandler.py

```diff
@@ -72,9 +97,11 @@ def __init__(self,
                  pass_job_queue=False,
                  pass_user_data=False,
                  pass_chat_data=False,
-                 allow_edited=False,
                  message_updates=True,
```

Comment:
It was in that order very very much for a reason. The other ones are less important, but this one could very well mess up backwards compat for users. @tsnoam thoughts?